src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1387,7 +1396,7 @@ pre.rust {
 #copy-path {
 	height: 34px;
 }
-#settings-menu > a, #help-button, #copy-path {
+#settings-menu > a, #help-button > button, #copy-path {
```

Comment:
Just in case: it's not one selector but three different selectors on which we apply the same rules. If it was what you understood, I'm not sure to understand how simpler it could be though...